### ISSUE TITLE
Setting size of points

### ISSUE THREAD

**Issue by vascotenner** [ioam/holoviews#281]
I want to have my markers larger in a plot:
```
%%opts Points (size=20)
hv.Points((0,0))
```
# Matplotlib
```
Traceback (most recent call last):
  File "/home/tenner/src/holoviews/holoviews/ipython/display_hooks.py", line 158, in wrapped
    widget_mode = widget_mode)
  File "/home/tenner/src/holoviews/holoviews/ipython/display_hooks.py", line 193, in element_display
    return render_plot(element_plot, False)
  File "/home/tenner/src/holoviews/holoviews/ipython/display_hooks.py", line 133, in render_plot
    plot.update(0)
  File "/home/tenner/src/holoviews/holoviews/plotting/mpl/plot.py", line 192, in update
    return self.initialize_plot()
  File "/home/tenner/src/holoviews/holoviews/plotting/mpl/chart.py", line 566, in initialize_plot
    edgecolors=edgecolor, **style)
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/axes/_axes.py", line 3649, in scatter
    collection.update(kwargs)
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/artist.py", line 757, in update
    raise AttributeError('Unknown property %s' % k)
AttributeError: Unknown property size
```
# Bokeh
```
Traceback (most recent call last):
  File "/home/tenner/src/holoviews/holoviews/ipython/display_hooks.py", line 159, in wrapped
    notebook_archive.add(element, html=html)
  File "/home/tenner/src/holoviews/holoviews/ipython/archive.py", line 186, in add
    info=dict(info, notebook=self.notebook_name))
  File "/home/tenner/src/holoviews/holoviews/core/io.py", line 666, in add
    rendered = exporter(obj)
  File "/home/tenner/src/holoviews/holoviews/plotting/mpl/renderer.py", line 88, in __call__
    plot, fmt =  self._validate(obj, fmt)
  File "/home/tenner/src/holoviews/holoviews/plotting/renderer.py", line 117, in _validate
    plot = self.plotting_class(obj)(obj, **self.plot_options(obj, self.size))
  File "/home/tenner/src/holoviews/holoviews/plotting/mpl/renderer.py", line 120, in plot_options
    options = Store.lookup_options(cls.backend, obj, 'plot').options
  File "/home/tenner/src/holoviews/holoviews/core/options.py", line 863, in lookup_options
    raise KeyError("No custom settings defined for object with id %d" % obj.id)
KeyError: 'No custom settings defined for object with id 0'
```

**Comment by jlstevens** [comment]
Annoying matplotlib has multiple aliases for the same thing which can be tricky for us to support and some of them might be tab-completing in the ``%opts`` magic. Try using ``%%opts Points (s=20)`` instead.

**Comment by jlstevens** [comment]
This particular example is fixed in 237e8f2. Matplotlib can be erratic regarding which style aliases are supported and which aren't. For this reason, invalid options need to be disable per plotting class (so we don't tab-complete suggestions that don't work).

So if anyone finds other examples of style options that tab complete but don't actually work, please inform us so we can disable them.

**Comment by vascotenner** [comment]
Thank you very much for all the good work for this nice module!
